spec/controllers/api/v2/application_controller_spec.rb

```diff
@@ -2,107 +2,42 @@
 
 describe API::V2::ApplicationController, type: :controller do
   describe '#authenticate' do
-    let(:user) { create(:user) }
+    let(:user)    { create(:user) }
     let(:payload) { { email: user.email } }
     let(:encoded_token) do
-      JWT.encode payload,
-                 Settings.authentication.secret,
-                 Settings.authentication.algorithm
+      JWT.encode(
+        payload,
+        Settings.authentication.secret,
+        Settings.authentication.algorithm
+      )
     end
     let(:bearer_token) { "Bearer #{encoded_token}" }
 
     before do
-      controller.response = response
+      controller.response              = response
       request.headers['Authorization'] = bearer_token
     end
 
     subject { controller.authenticate }
 
-    it 'saves the user for use by the action' do
-      controller.authenticate
-
-      expect(assigns(:current_user)).to eq user
-    end
-
-    context 'algorithm is not plain-text' do
-      it { should be true }
-
+    context 'successful authentication' do
```

Comment:
Sorry to keep banging on about this, but `context` should describe a setup or state, not an outcome. So here, I think "user is valid" or something similar would better the describe the state where we would expect "it saves the user for use by the action".